contracts/Controlled.sol

```diff
@@ -5,7 +5,7 @@ contract Controlled {
     /// @notice The address of the controller is the only address that can call
     ///  a function with this modifier
     modifier onlyController() {
-        require(msg.sender == controller);
+        require(msg.sender == controller, "Not authorized");
```

Comment:
If we are making changes here, then this would be a good opportunity to use custom errors instead.

String messages in revert are expensive. We can reduce this to fixed-size for bytes using custom error selectors.
Maybe something we can put into our gas-optimization analysis for this source.

Reply:
Agreed, but for fixing lint issues lets do the smallest change possible. We can do another PR replacing these strings with custom errors.

Reply:
Tracked at #11

Reply:
Absolutely!